website/docs/proxy/using-a-proxy-requiring-a-custom-ca.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 2
+title: Custom Certificate Authorities
+description: Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA).
+tags: [podman-desktop, installing, windows, macos, linux]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
+---
+
+# Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA) on macOS and Windows
+
+You can configure Podman Desktop to run behind a proxy requiring custom Certificate Authorities (CA), such as self-signed certificates.
+The configuration applies to the Podman engine, but not to your containers.
+
+#### Prerequisites
+
+* Your proxy URL: `<your.proxy.tld:port>`
+* Your proxy Certificate Authorities (CA) is stored in your home directory, in the `proxy_ca.pem` file, in Privacy-Enhanced Mail (PEM) format.
```

Comment:
I would add that this certificate file can have multiple extensions (cer, crt, pem?) so it is not confused (as I was) with also a private key extension (pem).

Reply:
The idea here is not to document all possible cases, but that the user does the conversion to have the certificate in PEM format.